.github/workflows/stale.yml

```diff
@@ -29,3 +29,15 @@ jobs:
           exempt-all-milestones: true
           operations-per-run: 500
           ascending: true
+
+      - name: Close Inactive Issues That Awaiting Author's Response
+        uses: actions/stale@v8
+        with:
+          days-before-stale: 3
```

Comment:
Does this mean when an issue is marked to be `'waiting-for: author'` for 3 days without a response, it will be markedas `'inactive'`? How long will it be closed after adding `'inactive'`? Can you link the PR or commit from ECharts bot?

Reply:
The echarts bot will immediately close the issue after it's labeled `inactive`. This is a present function of echarts-bot and has been used for a long time.
See also https://github.com/apache/echarts/issues?q=is%3Aissue+label%3Ainactive+is%3Aclosed

Reply:
If so, I think 3 days is too short. How about 14 days?

Reply:
I don't think it's worth waiting for two weeks.

Reply:
Not every developer opens GitHub every day, and even open the notification panel. We may discuss with other committers if we can't agree on this. :)